metadata/command.xml

```diff
@@ -7,9 +7,14 @@
 		<group>
 			<_short>Commands</_short>
 			<_long>Sets the command bindings.</_long>
+			<option name="binding" type="dynamic_list">
+				<_short>Binding</_short>
+				<_long>Sets the binding.</_long>
+				<type>activator</type>
+			</option>
 			<option name="command" type="dynamic_list">
```

Comment:
How do you expect to have more than one dynamic_list in a plugin with this syntax? The dynamic_list options should be grouped in some way.